api/mesh/v1alpha1/mesh/schema.yaml

```diff
@@ -0,0 +1,320 @@
+properties:
+  labels:
+    additionalProperties:
+      type: string
+    type: object
+  mesh:
+    type: string
```

Comment:
This should be there as it's a Global entity

Reply:
yes, scope change fixed that